test/test_collection.py

```diff
@@ -217,6 +231,29 @@ def test_imports(self):
         from pymongoexplain import ExplainableCollection
         self.assertEqual(ExplainableCollection, ExplainCollection)
 
+    def test_verbosity(self):
+        res = self.explain.find({})
+        self.assertFalse(self._recursiveIn("executionStats", res))
+        self.assertFalse(self._recursiveIn("allPlansExecution", res))
+        self.explain.update_settings(verbosity="executionStats")
+        res = self.explain.find({})
+        self.assertTrue(self._recursiveIn("executionStats", res))
+        self.assertFalse(self._recursiveIn("allPlansExecution", res))
+        self.explain.update_settings(verbosity="allPlansExecution")
+        res = self.explain.find({})
+        self.assertTrue(self._recursiveIn("executionStats", res))
+        self.assertTrue(self._recursiveIn("allPlansExecution", res))
+
+    def test_comment(self):
```

Comment:
Instead of checking the explain result, this test should probably assert that we sent the comment field with the explain comment using a CommandListener.

Reply:
Done.